Docs/source/usage/parameters.rst

```diff
@@ -1919,6 +1919,22 @@ In-situ capabilities can be used by turning on Sensei or Ascent (provided they a
     Default is ``<diag_name>.fields_to_plot = Ex Ey Ez Bx By Bz jx jy jz``.
     Note that the fields are averaged on the cell centers before they are written to file.
 
+* ``<diag_name>.particle_fields_to_plot`` (list of `strings`, optional)
+   Names of per-cell averages of particle properties to calculate and output. Default is none. Parser functions for these field names are specified by ``<diag_name>.particle_fields.<field_name>(x,y,z,ux,uy,uz)``.
+
+* ``<diag_name>.particle_fields_species`` (list of `strings`, optional)
+         Species for which to calculate ``particle_fields_to_plot``.
+         The default is a list of all of the available particle species.
```

Comment:
It's not clear to me from the docs yet if that means every field adds all species together or if this is one species per particle-derived field.
I think it's the latter from the implementation? Can you please specify that here?

Reply:
It is the latter -- I've updated the docs.